silx/gui/plot/actions/io.py

```diff
@@ -220,7 +224,8 @@ def _getAxesLabels(self, item):
         ylabel = item.getYLabel() or self.plot.getYAxis().getLabel()
         return xlabel, ylabel
 
-    def _selectWriteableOutputGroup(self, filename):
+    @staticmethod
+    def _selectWriteableOutputGroup(filename, parent):
```

Comment:
Same as about, adding `parent=None` avoids a few changes and allow only to set it when it matters.